src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -0,0 +1,716 @@
+  experimental: {
+    fonts: [{
+      provider: fontProviders.google(),
+      name: "Roboto",
+      cssVariable: "--font-roboto",
+    }]
+  }
+})
+```
+
+#### provider
+
+<p>
+
+**Type:** `AstroFontProvider | "local"`
+</p>
+
+The source of your font files. You can use a [built-in provider](#available-remote-font-providers), write your own [custom provider](#build-your-own-font-provider), or set to `"local"` to use local font files:
+
+```js title="astro.config.mjs" {6}
+import { defineConfig, fontProviders } from "astro/config"
+
+export default defineConfig({
+  experimental: {
+    fonts: [{
+      provider: fontProviders.google(),
+      name: "Roboto",
+      cssVariable: "--font-roboto"
+    }]
+  }
+})
+```
+
+#### name
+
+<p>
+
+**Type:** `string`
+</p>
+
+The font family name, as identified by your font provider:
+
+```js
+name: "Roboto"
+```
+
+#### cssVariable
+
+<p>
+
+**Type:** `string`
+</p>
+
+A valid [ident](https://developer.mozilla.org/en-US/docs/Web/CSS/ident) of your choosing in the form of a CSS variable (i.e. starting with `--`):
+
+```js
+cssVariable: "--font-roboto"
+```
+
+#### fallbacks
+
+<p>
+
+**Type:** `string[]`<br />
+**Default:** `["sans-serif"]`
+</p>
+
+An array of fonts to use when your chosen font is unavailable, or loading. Fallback fonts will be chosen in the order listed. The first available font will be used:
+
+```js
+fallbacks: ["CustomFont", "serif"]
+```
+
+To disable fallback fonts completely, configure an empty array:
+
+```js
+fallbacks: []
+```
+
+If the last font in the `fallbacks` array is a [generic family name](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#generic-name), an [optimized fallback](https://developer.chrome.com/blog/font-fallbacks) using font metrics will be generated. To disable this optimization, set `optimizedFallbacks` to false.
+
+#### optimizedFallbacks
+
+<p>
+
+**Type:** `boolean`<br />
+**Default:** `true`
+</p>
+
+Whether or not to enable optimized fallback generation. You may disable this default optimization to have full control over [`fallbacks`](#fallbacks):
+
+```js
+optimizedFallbacks: false
+```
+
+### Remote font properties
+
+Further configuration options are available for remote fonts. Set these to customize the data loaded from your [font provider](#available-remote-font-providers), for example to only download certain font weights or styles.
+
+Under the hood, these options are handled by [unifont](https://github.com/unjs/unifont/). Some properties may not be supported by some providers, and may be handled differently by each provider.
+
+#### weights
+
+<p>
+
+**Type:** `(number | string)[]`<br />
+**Default:** `[400]`
+</p>
+
+An array of [font weights](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight). If no value is specified in your configuration, only weight `400` is included by default to prevent unnecessary downloads. Include this array to download additional font weights:
+
+```js
+weights: [200, "300", "bold"]
```

Comment:
```suggestion
weights: [200, "400", "bold"]
```
I don't want it to look like they "get 400 automatically, even if they don't list it" because it happens to be the default!